brands/shop/hardware.json

```diff
@@ -83,6 +83,20 @@
         "shop": "hardware"
       }
     },
+    {
+      "displayName": "Robert Dyas",
+      "id": "robertdyas-6506bb",
```

Comment:
Do I need to change this ID at all?

Reply:
No need, the build script will take care of it.